fastlane/Fastfile

```diff
@@ -151,6 +151,36 @@ def get_changelog
   end
 end
 
+def prompt_for_release_notes ( instructions = "", reset_git_repo_on_fail = false)
```

Comment:
FWIW, argument defaults like this lead to hard-to-understand function calls like `prompt_for_release_notes( "", true )`: what are those arguments doing? Instead, the Ruby convention is for requirements to go in arguments and for multiple options to go in a final argument that's a hash, so more like 

```ruby
def prompt_for_release_notes( options = {} )
  instructions = options[:instructions] || ""
  if options[:reset_git_repo_on_fail]
    # do something special
  end
  # do other stuff
end
```

Then the call is much clearer:

```ruby
# do the default behavior
prompt_for_release_notes

# prompt with custom instructions
prompt_for_release_notes( instructions: "These are the special notes" )

# prompt and reset git on fail
prompt_for_release_notes( reset_git_repo_on_fail: true )
```

When there's just a single option and no requirements, sometimes a default arg is ok, like `get_changelog`, but even there the call would be more readable with named hash keys.

Reply:
Have refactored the arguments to a hash